tests/integration/test_integration.py

```diff
@@ -162,100 +129,86 @@ async def get_traefik_ip_address(ops_test: OpsTest) -> str:
     endpoints = await get_traefik_proxied_endpoints(ops_test)
     return _get_host_from_url(endpoints[TRAEFIK_CHARM_NAME]["url"])
 
+
 async def get_sdcore_nms_endpoint(ops_test: OpsTest) -> str:
-    """Retrieve the SD-Core NMS endpoint by using Traefik's `show-proxied-endpoints` action."""
     endpoints = await get_traefik_proxied_endpoints(ops_test)
     return endpoints[APP_NAME]["url"]
 
+
 def _get_host_from_url(url: str) -> str:
     """Return the host from a URL formatted as http://<host>:<port>/ or as http://<host>/."""
     return url.split("//")[1].split(":")[0].split("/")[0]
 
+
 def ui_is_running(nms_endpoint: str) -> bool:
-    """Return whether the UI is running."""
-    url = f"{nms_endpoint}network-configuration"
+    url = f"{nms_endpoint}config/v1/network-slice"
```

Comment:
to change back to f"{nms_endpoint}network-configuration" once the new image is used